packages/admin-ui-docs/docs/components/heading.mdx

```diff
@@ -34,8 +61,13 @@ import { Heading } from '@vtex/admin-ui'
 
 ## Props
 
-Besides the props listed bellow, you can use all the props accepted by the `h` JSX element.
+### Heading
+
+All props of `h` jsx element.
+
+### HeadingLevel
 
-| Name | Type                  | Description       | Required | Default |
-| ---- | --------------------- | ----------------- | -------- | ------- |
-| as   | `React.ComponentType` | Element to render | 🚫       | `'h1'`  |
+| Name     | Type                         | Description    | Required | Default |
+| -------- | ---------------------------- | -------------- | -------- | ------- |
+| level    | `1 or 2 or 3 or 4 or 5 or 6` | Optional level | 🚫       | -       |
```

Comment:
Is this going to allow the behaviour you were trying to avoid? Of a `h4` followed by an `h1`

Reply:
Yes! But it is a hatch-scape.